src/main/java/io/kokuwa/maven/helm/InstallMojo.java

```diff
@@ -11,19 +11,22 @@
 import lombok.Setter;
 
 /**
- * Mojo for simulate a template.
+ * Mojo for executing "helm install".
  *
  * @author Tim IJntema
+ * @see "https://helm.sh/docs/helm/helm_install"
  * @since 07.02.2021
  */
 @Mojo(name = "install", defaultPhase = LifecyclePhase.DEPLOY, threadSafe = true)
 @Setter
 public class InstallMojo extends AbstractHelmWithValueOverrideMojo {
 
+	/** Helm command to execute. */
 	@Deprecated // java8 (since = "6.5.0", forRemoval = true)
```

Comment:
Please preserve both javadoc `@deprecated`  tag with description and `@Deprecated` annotation